xla/service/gpu/runtime/command_buffer_cmd.cc

```diff
@@ -87,6 +88,17 @@ limitations under the License.
 
 namespace xla::gpu {
 
+namespace {
+std::optional<se::DeviceMemoryBase> AssignBufferIfNotNull(
```

Comment:
instead of anonymous namespace can you make it static, and move next to `FusedMHACmd` as it doesn't have any other users, or maybe make it a static member of `FusedMHACmd`

Reply:
This will also be used by future FusedMHABackward command, looks like put here is better here.